doc/articles/common-issues-ioscatalyst.md

```diff
@@ -19,3 +19,13 @@ In order to fix this, add the following to your csproj (Xamarin, `net6.0-ios`, `
   <MtouchExtraArgs>$(MtouchExtraArgs) --registrar=static</MtouchExtraArgs>
 </PropertyGroup>
 ```
+## Error while retrieving IOS device in VS code
+
+When switching to an iOS debugging target in VS Code, you might encounter an error stating that the iOS device could not be retrieved. The error message may appear as follows:
+
+```
+[Info]: Project reload forced to switch to net8.0-ios | Debug
+[Error] Could not retrieve ios devices within 10 seconds. Aborting...
```

Comment:
```suggestion
[Error]: Could not retrieve ios devices within 10 seconds. Aborting...
```

Reply:
no, that's the output of the extension - it should not be different in the documentation